source/_components/light.mqtt_template.markdown

```diff
@@ -2,22 +2,22 @@
 layout: page
 title: "MQTT Template Light"
 description: "Instructions for how to setup MQTT Template lights within Home Assistant."
-date: 2016-11-06 21:16
+date: 2016-11-23 23:14
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: mqtt.png
 ha_category: Light
 ha_iot_class: "Local Push"
-ha_release: 0.33
+ha_release: 0.34
```

Comment:
`ha_release` refers to the version of Home Assistant that the component was introduced. Since this is just an addition to a component there's no need to change this.